content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1052 @@
+
+* bevy_sprites: @cart, @StarArawn 
+* bevy_pbr: Rob Swain (@superdump), @aevyrie, @cart, @zicklag, @jakobhellermann
+* bevy_ui: @Davier
+* bevy_text: @Davier
+* bevy_gltf: Rob Swain (@superdump)
+
+### WebGL2 Support
+
+<div class="release-feature-authors">authors: @zicklag, @mrk-its, @mockersf, Rob Swain (@superdump)</div>
+
+Bevy now has built-in support for deploying to the web using WebGL2 / WASM, thanks to @zicklag adding a native WebGL2 backend to wgpu. There is now no need for the third party `bevy_webgl2` plugin. Any Bevy app can be deployed to the web by running the following commands:
+
+```rust
+cargo build --target wasm32-unknown-unknown
+wasm-bindgen --out-dir OUTPUT_DIR --target web TARGET_DIR
+```
+
+The New Bevy Renderer developers prioritized cross-platform compatibility for the initial renderer feature implementation and so had to carefully operate within the limits of WebGL2 (ex: storage buffers and compute shaders aren't supported in WebGL2), but the results were worth it! Over time, features will be implemented that leverage more modern/advanced features such as compute shaders. But it is important to us that everyone has access to a solid visual experience for their games and applications regardless of their target platform(s).
+
+You can try out Bevy's WASM support in your browser using our new [Bevy Examples](/examples) page:
+
+[![wasm bevy examples](bevy_examples_wasm.png)](/examples)
+
+### Infinite Reverse Z Perspective Projection
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+For improved precision in the "useful range", the industry has largely adopted "reverse projections" with an "infinite" far plane. The new Bevy renderer was adapted to use the "right handed infinite reverse z" projection. [This Nvidia article](https://developer.nvidia.com/content/depth-precision-visualized) does a great job of explaining why this is so worthwhile. 
+
+### Compute Shaders
+
+The new renderer makes it possible for users to write compute shaders. Our new ["compute shader game of life" example](https://github.com/bevyengine/bevy/blob/v0.6.0/examples/shader/compute_shader_game_of_life.rs) (by @jakobhellermann) illustrates how to write compute shaders in Bevy.
+
+![compute game of life](compute.png)
+
+
+### New Multiple Windows Example
+
+<div class="release-feature-authors">authors: @DJMcNab</div>
+
+The "multiple windows" example has been updated to use the new renderer APIs. Thanks to the new renderer APIs, this example is now [much nicer to look at](https://github.com/bevyengine/bevy/blob/v0.6.0/examples/window/multiple_windows.rs) (and will look even nicer when we add high level Render Targets).
+
+![multiple windows](multiple_windows.png)
+
+### Crevice
+
+<div class="release-feature-authors">authors: @cart, @mockersf, Rob Swain (@superdump)</div>
```

Comment:
Maybe we should also credit ElectronicRU for the sized arrays PR? https://github.com/LPGhatguy/crevice/pull/27

Reply:
Oh, you did already below.